release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2058,6 +2058,87 @@ $ oc annotate -n <NAMESPACE> route/<ROUTE-NAME> "haproxy.router.openshift.io/bal
+
+* A clock jump error occurs in the `linuxptp` subsystem that is deployed by the PTP Operator. The reported error message is: `clock jumped backward or running slower than expected!`. The error is encountered in a host with an Intel Columbiaville E810 NIC installed in a {product-title} version 4.8 or 4.9 cluster. The error is likely Intel ice driver related, rather than an error in the `linuxptp` subsystem. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013478[*BZ#2013478*])
+
+* Sometimes Operator installation fails during zero touch provisioning (ZTP) installation of a DU node. The `InstallPlan` API reports an error. The reported error message is: `Bundle unpacking failed. Reason: DeadlineExceeded`. The error occurs if the Operator installation job exceeds 600 seconds.
++
+To workaround this issue and re-try the Operator install, run the following `oc` commands for the failed Operator:
++
+. Delete the catalog source:
++
+[source,terminal]
+----
+$ oc -n openshift-marketplace delete catsrc <failed_operator_name>
+----
++
+. Delete the install plan:
++
+[source,terminal]
+----
+$ oc -n <failed_operator_namespace> delete ip <failed_operator_install_plan>
+----
++
+. Delete the subscription and wait for the Operator `CatalogSource` and `Subscription` resources to be re-created by the relevant custom resource policy:
++
+[source,terminal]
+----
+$ oc -n <failed_operator_namespace> delete sub <failed_operator_subscription>
+----
++
+.Expected result
++
+The Operator `InstallPlan` and `ClusterServiceVersion` resources are created and the Operator is installed.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021456[*BZ#2021456*])
+
+* Sometimes when the ZTP installation process finishes provisioning a DU node, the performance profile configuration is not applied to the node and the `MachineConfigPool` resource becomes stuck in an `Updating` state.
++
+To workaround this issue, perform the following procedure:
++
+. Get the name of the failed DU node:
++
+[source,terminal]
+----
+$ oc get mcp
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME              CONFIG                                   UPDATED   UPDATING   DEGRADED
+control-plane-1   rendered-control-plane-1-90fe2b00c718    False     True       False
+compute-1         rendered-compute-1-31197fc6da09          True      False      False
+----
++
+. Uncordon the failed node, and wait for the `machine-config-daemon` to apply the performance profile. For example:
++
+[source,terminal]
+----
+$ oc adm uncordon compute-compute-1-31197fc6da09
+----
++
+.Expected result
++
+The `machine-config-daemon` applies the performance profile configuration to the node.
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021151[*BZ#2021151*])
```

Comment:
Suggested updates after 4.9 GA review today:

* A race condition exists between the SRIOV operator and the MCO which can manifest itself in different ways.  Sometimes when the ZTP installation process finishes provisioning a DU node, the performance profile configuration is not applied to the node and the `MachineConfigPool` resource becomes stuck in an `Updating` state.
+
To workaround this issue, perform the following procedure:
+
. Get the name of the failed DU node:
+
[source,terminal]
----
$ oc get mcp
----
+
.Example output
+
[source,terminal]
----
NAME              CONFIG                                   UPDATED   UPDATING   DEGRADED
control-plane-1   rendered-control-plane-1-90fe2b00c718    False     True       False
compute-1         rendered-compute-1-31197fc6da09          True      False      False
----
+
. Uncordon the failed node, and wait for the `machine-config-daemon` to apply the performance profile. For example:
+
[source,terminal]
----
$ oc adm uncordon compute-compute-1-31197fc6da09
----
+
.Expected result
+
The `machine-config-daemon` applies the performance profile configuration to the node.
+
Another manifestation of this issue is that the performance profile configuration does not get applied during DU node configuration.  To work around this issue:
+
. Apply rpm-ostree kargs manually:
+
[source,terminal]
----
$ rpm-ostree  kargs --append=skew_tick=1 --append=nohz=on --append=rcu_nocbs=2-23,26-47 --append=tuned.non_isolcpus=03000003 --append=intel_pstate=disable --append=nosoftlockup --append=tsc=nowatchdog --append=intel_iommu=on --append=iommu=pt --append=isolcpus=managed_irq,2-23,26-47 --append=systemd.cpu_affinity=0,1,24,25 --append=default_hugepagesz=1G --append=hugepagesz=1G --append=hugepages=32 --append=idle=poll --append=rcupdate.rcu_normal_after_boot=0 --append=nohz_full=2-23,26-47
----
+
. Reboot the node
+
.Expected result
+
The Performance profile is applied properly.
+
Another way this issue manifests itself is that the DU node reboot during policy configuration can take tens of minutes.  Note that no workaround is required here.  The system will eventually recover.
+
(link:https://bugzilla.redhat.com/show_bug.cgi?id=2021151[*BZ#2021151*])

Reply:
/lgtm